assets/js/current-projects.js

```diff
@@ -246,6 +268,40 @@ function cancelMobileFiltersEventHandler(e) {
     hideFiltersEventHandler(e)
     clearAllEventHandler()
 }
+//search bar event handler
+function searchEventHandler(){
+    event.preventDefault();
+    let searchTerm=document.querySelector("#search").value;
+    let tokenObj={};
+    tokenObj['Search']=searchTerm;
+     
+   const filterParams = Object.fromEntries(new URLSearchParams(window.location.search));
+    if (Object.keys(filterParams).length>0) {    
+        for(const [key,value] of Object.entries(filterParams)){
+            if (key !== 'Search'){
+                tokenObj[key]=value.split(',');
+            }
+        }  
+    }
+    let queryString = Object.keys(tokenObj).map(key => key + '=' + tokenObj[key]).join('&').replaceAll(" ","+");
+
+    window.history.replaceState(null,'',`?${queryString}`)
+}
+
+function searchEnterKeyHandler(e){
+    if (event.key === "Enter") {
+        document.querySelector(".search-glass").click();
+    }
+}
+
+function searchOnFocusEventHandler(){
+    document.querySelector(".search-x").style.display='block';
+}
+
+function searchCloseEventHandler(){
```

Comment:
Same here. The [event property](https://developer.mozilla.org/en-US/docs/Web/API/Window/event) is deprecated.
```suggestion
function searchCloseEventHandler(e){
e.preventDefault();
```